src/frontend/nginx.conf

```diff
@@ -1,32 +1,25 @@
-events {
-  worker_connections  256;
-}
+worker_processes 4;
+
+events { worker_connections 1024; }
 
 http {
     include   /etc/nginx/mime.types; # fix it as per your env
```

Comment:
for docker, we need this? we should not say `# fix it as per your env` for docker

Reply:
let's have 2 conf 10 `nginx.conf` and `nginx-docker.conf` should have very minimal difference to make it work